ch390/src/esp_eth_phy_ch390.c

```diff
@@ -41,9 +53,10 @@ typedef union {
     };
     uint32_t val;
 } phy_ctl1_reg_t;
```

Comment:
Is this register description available only CH182**DS2** datasheet? I cannot find it in DS1 which is only available on their website? Is it possible to obtain CH182**DS2** datasheet somewhere?

Reply:
Yes～I asked WCH technicians for ch182ds2 and they sent it to me via email and told me not to disclose its entire contents(it is okay to disclose the part, so I just wrote the definition of a register). But to be honest, I am really confused and worried. I wonder if there will be any legal risks😰😰😰~ If you really need this datasheet, you could either send an email to WCH on their official website, or send an email to me in private(addr is on my homepage, but please promise me not to let it spread).

Reply:
OK, thank you for the info. I understand, it'll be better if you not share with me. I'll ask them by myself. Is it in English?

Reply:
> OK, thank you for the info. I understand, it'll be better if you not share with me. I'll ask them by myself. Is it in English?

Nop~😢

Reply:
I contacted WCH. They didn't share the datasheet but I consulted the matter with one of their representatives. Please see the comment below.